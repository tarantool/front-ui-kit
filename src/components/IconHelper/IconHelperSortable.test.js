import React from 'react';
import renderer from 'react-test-renderer';
import { IconHelperSortable } from './index';

it('IconHelperSortable renders correctly', () => {
  const tree = renderer
    .create(
      <>
        <IconHelperSortable sort="asc" />
        <IconHelperSortable sort="desc" />
        <IconHelperSortable />
      </>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
