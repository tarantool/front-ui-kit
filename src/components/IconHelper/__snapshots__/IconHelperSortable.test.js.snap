// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconHelperSortable renders correctly 1`] = `
Array [
  .emotion-0 {
  width: 16px;
  height: 16px;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  vertical-align: middle;
  width: 14px;
  height: 14px;
  width: 16px;
  height: 16px;
  fill: #5c606c;
  -webkit-transform: rotateX(180deg);
  -moz-transform: rotateX(180deg);
  -ms-transform: rotateX(180deg);
  transform: rotateX(180deg);
}

<svg
    className="emotion-0"
    viewBox="0 0 16 16"
  >
    <use
      xlinkHref="#icon-sort-desc"
    />
  </svg>,
  .emotion-0 {
  width: 16px;
  height: 16px;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  vertical-align: middle;
  width: 14px;
  height: 14px;
  width: 16px;
  height: 16px;
  fill: #5c606c;
}

<svg
    className="emotion-0"
    viewBox="0 0 16 16"
  >
    <use
      xlinkHref="#icon-sort-desc"
    />
  </svg>,
  .emotion-0 {
  width: 16px;
  height: 16px;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  vertical-align: middle;
  width: 14px;
  height: 14px;
  width: 16px;
  height: 16px;
  fill: #5c606c;
  -webkit-transform: rotateX(180deg);
  -moz-transform: rotateX(180deg);
  -ms-transform: rotateX(180deg);
  transform: rotateX(180deg);
  fill: #e6e7e8!important;
}

<svg
    className="emotion-0"
    viewBox="0 0 16 16"
  >
    <use
      xlinkHref="#icon-sort-desc"
    />
  </svg>,
]
`;
